core/server/services/invitations/accept.js

```diff
@@ -18,6 +18,15 @@ async function accept(invitation) {
         throw new errors.NotFoundError({message: i18n.t('errors.api.invites.inviteExpired')});
     }
 
+    let user = await models.User.findOne({email: data.email});
+    if (user) {
+        throw new errors.ValidationError({
+            message: i18n.t('errors.api.invites.inviteEmailAlreadyExist.message'),
+            context: i18n.t('errors.api.invites.inviteEmailAlreadyExist.context'),
+            errorDetails: i18n.t('errors.api.invites.inviteEmailAlreadyExist.help')
```

Comment:
```suggestion
            help: i18n.t('errors.api.invites.inviteEmailAlreadyExist.help')
```

The convention is to use `help` property when throwing an error to provide information on how the end user could fix the problem :wink:

Reply:
@naz sure , i can make this change. But you are aware that this change will only be visible on the server console and end user would only be shown the string in message (i.e `message: i18n.t('errors.api.invites.inviteEmailAlreadyExist.message')`) . right?

Reply:
Yes, displaying a `message` part in the UI is the convention we have right now for most errors like this one. Again, might be changed in the future, but would need to be as a part of a global "error messages revamp". Breaking existing convention for the sake of single case will make it harder to refactor and reason about in the future :)
 
// cc @peterzimon, context above might be useful when reviewing [the copy](https://github.com/TryGhost/Ghost/issues/12060#issuecomment-684178756) for this case

Reply:
@naz agreed, pushed the changes. Thanks